### PR TITLE
Delete update status HostZoneUnbonding_EXIT_TRANSFER_QUEUE in UpdateHostZoneUnbondings function

### PR DESCRIPTION
**Pull request by ThanhNhann**
Closes: #525

## Context and purpose of the change

`HostZoneUnbonding`'s status `EXIT_TRANSFER_QUEUE` was set twice, [before](https://github.com/Stride-Labs/stride/blob/v2.0.3/x/stakeibc/keeper/icacallbacks_undelegate.go#L95) and [after](https://github.com/Stride-Labs/stride/blob/v2.0.3/x/stakeibc/keeper/icacallbacks_undelegate.go#L106) burning `stTokens` during `UndelegateCallback` so I think it should be used in `SetHostZoneUnbondings` instead of `UpdateHostZoneUnbondings` function because there is a param type of `recordstypes` in `SetHostZoneUnbondings` and the main reason is after Updatehostzone we have to set it to store

## Brief Changelog

- Delete update status of `hostZoneUnbonding` in `UpdateHostZoneUnbondings` function

## Author's Checklist

I have...

- [ ] Run and PASSED locally all GAIA integration tests
- [ ] If the change is contentful, I either:
    - [ ] Added a new unit test OR 
    - [ ] Added test cases to existing unit tests
- [ ] OR this change is a trivial rework / code cleanup without any test coverage

If skipped any of the tests above, explain.

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] manually tested (if applicable)
- [ ] confirmed the author wrote unit tests for new logic
- [ ] reviewed documentation exists and is accurate

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? 
  - [ ] Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`?
  - [ ] This pull request updates existing proto field values (and require a backend and frontend migration)? 
  - [ ] Does this pull request change existing proto field names (and require a frontend migration)?
  How is the feature or change documented? 
      - [ ] not applicable
      - [ ] jira ticket `XXX` 
      - [ ] specification (`x/<module>/spec/`) 
      - [ ] README.md 
      - [ ] not documented

### CODE REVIEW

**Review by sampocs — APPROVED** [review]
lgtm, nice catch!

File: x/stakeibc/keeper/icacallbacks_undelegate.go [hunk 1]
```diff
@@ -40,14 +40,15 @@ func (k Keeper) UnmarshalUndelegateCallbackArgs(ctx sdk.Context, undelegateCallb
...
 // ICA Callback after undelegating
-//   If successful:
-//      * Updates epoch unbonding record status
-// 		* Records delegation changes on the host zone and validators,
-//      * Burns stTokens
-//   If timeout:
-//      * Does nothing
-//   If failure:
-//		* Reverts epoch unbonding record status
+//
```
[the remark this answers is not in the document]
Comment (ThanhNhann) [reply x/stakeibc/keeper/icacallbacks_undelegate.go:43]: This change from a linter of my vs code 👨‍💻, if you think it shouldn't change, I will revert it

**Comment by sampocs** [comment]
Open

**Comment by ThanhNhann** [comment]
hi @asalzmann, can you plz review and merge this pr?

**Pull request merged**